{"name": "test_learn_embibe_explainers", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'send_keys'", "trace": "self = <Test.test_E2E.TestEmbibe object at 0x106f4fe10>\n\n    @pytest.mark.usefixtures(\"setup\", \"log_on_failure\")\n    def test_learn_embibe_explainers(self):\n>       self.test_sign_in_password()\n\nTest/test_E2E.py:211: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTest/test_E2E.py:22: in test_sign_in_password\n    login.test_sign_in_password()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <PageObject.signUp_signIn.signIn_up object at 0x1070069d0>\n\n    def test_sign_in_password(self):\n        self.driver.find_element(*signIn_up.get_started).click()\n>       self.driver.find_element(*signIn_up.email_field).send_keys(self.mobile_num)\nE       AttributeError: 'NoneType' object has no attribute 'send_keys'\n\nPageObject/signUp_signIn.py:88: AttributeError"}, "start": 1747730890217, "stop": 1747730890518, "uuid": "93ed88a7-e46d-45ad-a2a8-5dbe2d5e47a7", "historyId": "3edb9087b1bc7882eec132f72c26ec7a", "testCaseId": "3edb9087b1bc7882eec132f72c26ec7a", "fullName": "Test.test_E2E.TestEmbibe#test_learn_embibe_explainers", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "Test"}, {"name": "suite", "value": "test_E2E"}, {"name": "subSuite", "value": "TestEmbibe"}, {"name": "host", "value": "LM0010069"}, {"name": "thread", "value": "4572-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test.test_E2E"}]}